pkg/server/handler/auth.go

```diff
@@ -580,8 +583,19 @@ func (h *PasswordHandler) Handle(payload *router.Payload, response *router.Respo
 		panic(err)
 	}
 
-	response.Result = AuthResponse{
-		UserID:      info.ID,
-		AccessToken: token.AccessToken,
+	user := payload.User
```

Comment:
how about using the require user preprocessor? instead of checking if the user exists here? If you keep the tracking here, you need to return a good error with code rather than panic (difference is 4XX and 500)